frontends/web/src/locales/en/app.json

```diff
@@ -25,6 +25,9 @@
         "start": "Get started by depositing some coins to the wallet or buying directly in the BitBoxApp."
       }
     },
+    "exchangeCTA": {
+      "exchange": "Exchange {{unit}}"
+    },
```

Comment:
duplicate of `exchange.exchangeCTA`?

Reply:
Ups good point - i forgot the remove the old one. Removed :)